### PR TITLE
#60: Restaurant results page

### PR DESCRIPTION
**Pull request by thomastai1666**
Creates the restaurant page. closes issue #60 

Screen Shot 2021-10-28 at 8 36 40 PM

Screen Shot 2021-10-28 at 8 36 21 PM

### CODE REVIEW

File: front-end/src/components/ResultsCell.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import React from "react";
+
+function ResultsCell(props) {
+  const { name, description, percentageMatch } = props;
+  return (
+    <div className="ResultsCell">
+      <div className="flex">
+        <div className="left">
+          <img src={restaurauntPlaceholder} alt=""></img>
```
Comment (yunko1803) [inline front-end/src/components/ResultsCell.jsx:12]: self closing tag

File: front-end/src/components/ResultsCell.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  border: 1px solid rgb(226, 226, 226);
+  box-shadow: 5px 5px 10px -6px rgba(0, 0, 0, 0.2);
+  border-radius: 10px;
+  margin: 10px;
+  height: 130px;
+}
+
+.ResultsCell img {
+  height: 130px;
+  border-radius: 10px 00px 0px 10px;
```
Comment (yunko1803) [inline front-end/src/components/ResultsCell.css:13]: 00 -> 0

Comment (thomastai1666) [reply front-end/src/components/ResultsCell.css:13]:
```suggestion
  border-radius: 10px 0px 0px 10px;
```

File: front-end/src/components/ResultsCell.jsx (hunk 1 above)
Comment (thomastai1666) [reply front-end/src/components/ResultsCell.jsx:12]:
```suggestion
          <img src={restaurauntPlaceholder} alt=""/>
```

**Pull request merged**